packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -247,6 +255,78 @@ export class AtomicCategoryFacet
     );
   }
 
+  public ariaLabelForSearchResult(searchResult: CategoryFacetSearchResult) {
```

Comment:
I kept it at the component level instead of creating a new class, which would require defining more interfaces similar to the FacetSearch